rcl/test/rcl/test_arguments.cpp

```diff
@@ -337,8 +396,9 @@ TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_remap) {
 }
 
 TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_one_remap_two_ros_args) {
-  const char * argv[] = {"process_name", "--ros-args", "--ros-args", "-r", "/foo/bar:=/fiz/buz"};
-  int argc = sizeof(argv) / sizeof(const char *);
+  const char * const argv[] =
+  {"process_name", "--ros-args", "--ros-args", "-r", "/foo/bar:=/fiz/buz"};
```

Comment:
This is a bit of a strange indentation (uncrustify is weird). Would the following pass?
```
const char * const argv[] = {
  "process_name", "--ros-args", "--ros-args", "-r", "/foo/bar:=/fiz/buz"};
```

There are a couple more instances of this below.

Reply:
I thought exactly the same when I added commit 8a1522a. The three changes made in that commit where failing the uncrustify test.